pseudo_code/linear_fit.py

```diff
@@ -0,0 +1,54 @@
+def linear_fit(data):
```

Comment:
I honestly think it may be easier to make the parameters for input in this function be x, y, sigma as lists of each, just to make it easier to work with. If you want to have the input be a csv file, this requires a bit more touchup such as using pandas to read the csv and extract values out of it.
This would require the csv the data is found in to have been read by a different function before using this function, so idk. Maybe @kylemasc917 can give his advice.
```suggestion
def linear_fit(x, y, sigma):
```